compiler/rustc_expand/src/tests.rs

```diff
@@ -148,20 +148,10 @@ fn test_harness(file_text: &str, span_labels: Vec<SpanLabel>, expected_output: &
             println!("text: {:?}", source_map.span_to_snippet(span));
         }
 
-        let emitter = EmitterWriter::new(
-            Box::new(Shared { data: output.clone() }),
-            Some(source_map.clone()),
-            None,
-            fallback_bundle,
-            false,
-            false,
-            false,
-            None,
-            false,
-            false,
-            TerminalUrl::No,
-        );
-        let handler = Handler::with_emitter(true, None, Box::new(emitter), None);
+        let emitter =
+            EmitterWriter::new(Box::new(Shared { data: output.clone() }), fallback_bundle, false)
+                .sm(Some(source_map.clone()));
```

Comment:
Surprised that `.sm(Some(source_map))` is needed here, instead of `.sm(source_map)`.

Reply:
In that case, r=me after you're done with timer